docs/streams/developer-guide/config-streams.html

```diff
@@ -326,8 +331,15 @@ <h4><a class="toc-backref" href="#id23">num.standby.replicas</a><a class="header
               the <code>org.apache.kafka.streams.state.DslStoreSuppliers</code> interface.
             </td>
             <td><code>BuiltInDslStoreSuppliers.RocksDBDslStoreSuppliers</code></td>
+            <td colspan="2">Default serializer/deserializer for the inner class of windowed keys, implementing the <code class="docutils literal"><span class="pre">Serde</span></code> interface.</td>
+            <td>null</td>
           </tr>
-          <tr class="row-even"><td>max.task.idle.ms</td>
+          <tr class="row-even"><td>default.windowed.value.serde.inner (Deprecated.)</td>
```

Comment:
It seems `default.windowed.value.serde.inner` exist already? Why do we add it here?